webapp/src/actions/index.js

```diff
@@ -27,6 +29,8 @@ export function getAttributes(userID = '') {
             return {error};
         }
 
+        (data || []).forEach((attribute) => dispatch(getCustomEmojisInText(attribute)));
```

Comment:
Minor nit: Could we initialize this when we declare `data` above for readability here?

Reply:
Ah, yes! For some reason, I thought `data` might still get written in the exceptional case, but you're right. This is much cleaner.